### DISCUSSION TITLE
请问这个登录逻辑是不是正常的？

### DISCUSSION THREAD

**Discussion by Arondight** [takayama-lily/oicq#283] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/13673702/147941885-bd87fca8-d8ca-43da-9dad-fa733b4cdfa6.png)

我不确定我写的密码登录和二维码登录中处理的事件是不是正确的，例如设备锁是否都要处理等等，不知道图中的逻辑是正常的吗？

**Comment by takayama-lily** [comment, marked as answer]
catpcha文字验证码事件已经没有了

**Reply by Arondight** [reply]
谢谢指导
